repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/statement/UpdateByIdStatement.java

```diff
@@ -25,11 +26,11 @@ public final class UpdateByIdStatement<ENTITY extends Entity<ENTITY>, ID extends
     private final Set<YqlStatementParam> idParams;
 
     public UpdateByIdStatement(Class<ENTITY> type, UpdateModel.ById<ID> model) {
-        this(type, model, EntitySchema.of(type).getName());
+        this(EntityDescriptor.of(type), model);
     }
 
-    public UpdateByIdStatement(Class<ENTITY> type, UpdateModel.ById<ID> model, String tableName) {
-        super(EntitySchema.of(type), EntitySchema.of(type), tableName);
+    public UpdateByIdStatement(EntityDescriptor<ENTITY> descriptor, UpdateModel.ById<ID> model) {
+        super(EntitySchema.of(descriptor.clazz()), EntitySchema.of(descriptor.clazz()), descriptor.getTableName(EntitySchema.of(descriptor.clazz())));
```

Comment:
`EntitySchema.of(descriptor.clazz())` -> `descriptor.toSchema()`?

(and let's also add `YqlStatement` constructor taking `EntityDescriptor`, *but in a separate PR*)